### PR TITLE
website: 2024.10 Release Notes

### PR DESCRIPTION
**Pull request by gergosimonyi**

### CODE REVIEW

File: website/docs/releases/2024/v2024.10.md [hunk 1]
```diff
@@ -9,14 +9,43 @@ slug: "/releases/2024.10"
...
+
+We have no breaking changes this release!
 
 ## New features
 
 -   **Invalidation flows for providers**
 
     The sign-out experience when the session in an application ends can be configured now. Previously where this was always a static page, any flow can be used now. This can be used for additional validation, or redirecting the user to a custom URL.
 
+-   **JWE support for OAuth**
```
Comment (fheisler) [inline website/docs/releases/2024/v2024.10.md:29]: swap to match above ordering

File: website/docs/releases/2024/v2024.10.md [hunk 2]
```diff
@@ -9,14 +9,43 @@ slug: "/releases/2024.10"
...
 :::::
 
-<!-- ## Breaking changes -->
+## Highlights
+
+-   **Chrome Device Trust** <span class="badge badge--primary">Enterprise</span> <span class="badge badge--info">Preview</span>: Verify that your users are logging in from managed devices and validate the devices' compliance with company policies.
+-   **FIPS/FAL3 for FedRAMP "very high" compliance** <span class="badge badge--primary">Enterprise+</span>: with support for SAML encryption and now JWE (JSON Web Encryption) support, authentik can now be configured for FIPS compliance at Federal Assurance Level (FAL) 3.
+-   **Captcha on Identification stage**: Run a CAPTCHA process in the background while the user is entering their identification.
+-   **Kerberos source**: authentik can now integrate with existing Kerberos environments by allowing users to log in with their Kerberos credentials or SPNEGO or syncing users into authentik.
```
Comment (BeryJu) [inline website/docs/releases/2024/v2024.10.md:17]:
```suggestion
-   **Kerberos source**: authentik can now integrate with existing Kerberos environments by allowing users to log in with their Kerberos credentials, SPNEGO or syncing users into authentik.
```

File: website/docs/releases/2024/v2024.10.md [hunk 3]
```diff
@@ -9,14 +9,43 @@ slug: "/releases/2024.10"
...
+
+## Breaking changes
+
+We have no breaking changes this release!
```
Comment (BeryJu) [inline website/docs/releases/2024/v2024.10.md:21]:
```suggestion
We have no breaking changes this release.
```

File: website/docs/releases/2024/v2024.10.md [hunk 4]
```diff
@@ -9,14 +9,43 @@ slug: "/releases/2024.10"
...
 
 ## New features
 
+-   **Chrome Device Trust** <span class="badge badge--primary">Enterprise</span> <span class="badge badge--info">Preview</span>
+
+    This is a new stage for Enterprise clients that verifies the user through the Chrome Verified Access API. This stage only works with Google Chrome. You'll need to bring your own [Verified Access API instance](https://developers.google.com/chrome/verified-access/overview) via Google Cloud.
+
+-   **JWE support for OAuth**
+
+    You can now configure JSON Web Encryption with the OAuth 2.0 Provider, which will encrypt all the tokens created by authentik.
```
Comment (BeryJu) [inline website/docs/releases/2024/v2024.10.md:31]:
```suggestion
    You can now configure JSON Web Encryption with the OAuth 2.0 Provider, which will encrypt all the tokens created by authentik, using the configured encryption key.
```

**Review by BeryJu — APPROVED** [review]
couple small changes

**Pull request merged**